makeservices/makemysql-real

```diff
@@ -155,85 +112,88 @@ def main():
             connection.store_result()
             # Result should be "Query OK, 0 rows affected",
             # but we'll assume no exception means success.
         except:
-            print('>>> Error: Failed to grant database privileges.')
-            sys.exit(1)
+            print('>>> Error: Failed to grant database privileges.\n')
+            exit(1)
 
         # Create new database, if necessary.
         if db_create:
             try:
-                print(">>> Creating new database '%s'..." % username)
-                query = CREATE_QUERY % username
+                print(">>> Creating new database '{}'...".format(username))
+                query = CREATE_QUERY.format(username)
                 connection.query(query)
                 connection.store_result()
                 # Result should be "Query OK, 1 row affected",
                 # but we'll assume no exception means success.
             except:
-                print('>>> Error: Failed to create database.')
-                sys.exit(1)
+                print('>>> Error: Failed to create database.\n')
+                exit(1)
 
-        # Database is ready for use. Send out a welcome email.
+        # Database is ready for use.
         if db_create:
-            print()
-            print('Your MySQL database has been created.')
-            print()
-            print('Please check your OCF email account momentarily for information')
-            print('regarding how to access and manage your MySQL database.')
-            print()
-            print('You can also find the same information at:')
-            print()
-            print('  http://ocf.berkeley.edu/docs/services/mysql/')
-            print()
-
-        print('>>> Your MySQL database password is: %s' % userpass)
-        print()
-    except SystemExit:
-        # Do not go gentle into that good night. Or not...
-        pass
+            print(dedent(
+                """\
+                Your MySQL database has been created.
+
+                Please check your OCF email account momentarily for information
+                regarding how to access and manage your MySQL database.
+
+                You can also find the same information at:
+
+                  http://ocf.berkeley.edu/docs/services/mysql/
+                """
+            ))
+
+        print('>>> Your MySQL database password is: {}\n'.format(userpass))
     except:
-        send_error_email(username, sys.exc_info())
-        print(sys.exc_info())
-        print()
-        print('A fatal error was detected during program execution.')
-        print('OCF Staff has been notified of the problem.')
-        print()
-
-#
-# Entry point.
-#
-main()
+        send_error_email(username, exc_info())
+        print(dedent(
+            """\
+            {}
+
+            A fatal error was detected during program execution.
+            OCF staff have been notified of the problem.
+            """
+        ).format(exc_info()))
```

Comment:
i don't think we want to print tracebacks to users, it's possible that the password might get leaked in them somehow. we need to be really careful with this script.